src/colors.rs

```diff
@@ -220,6 +220,57 @@ impl ColoredLevelConfig {
             Level::Trace => self.trace,
         }
     }
+
+    /// Returns the ansi escape sequence for the foreground color of the given log level.
+    pub fn get_ansi_fg(&self, level: &Level) -> &'static str {
```

Comment:
Awesome - this is even better!

Since we're adding this, do you think you could add a similar to `WithFgColor`? It could be something like:

```rust
/// An ansi escape color code.
struct AnsiEscape(&'static str)
```

If it implemented `fmt::Display` wrapping whatever's inside it with `"\x1B[{}m"` like this match statement, we could avoid having to enumerate all possible strings here. It'd be no extra runtime cost for slightly less code to maintain.

Reply:
I'm working on a nice solution for this, but it brings some changes which are out of the scope of this PR. I'll open a new one when I'm done ^-^
(might take a while though, I'm not feeling too well :/ )